cmake/options.cmake

```diff
@@ -85,6 +85,8 @@ option(OSQUERY_IGNORE_CMAKE_MAX_VERSION_CHECK "Ignore the maximum cmake version
 # Unfortunately, due glog always enabling BUILD_TESTING, we have to force it off, so that tests won't be built
 overwrite_cache_variable("BUILD_TESTING" "BOOL" "OFF")
 
+option(OSQUERY_BUILD_AWS "Whether to build the aws tables and library, to decrease memory usage and increase speed during build." ON)
```

Comment:
Nitpick, do you mind placing this next to the `OSQUERY_BUILD_BPF` so that we have all of the optional disables next to each other? I also see an inconsistency where we use `true` for BFP and `ON` for AWS.

Reply:
Not at all!
I'll change also the `OSQUERY_BUILD_BPF`; I prefer to keep ON/OFF since it's what one normally finds in the docs (although one can also use the other variations).